Resources/Prototypes/_EinsteinEngines/Entities/Modsuits/security-responsory.yml

```
@@ -0,0 +1,160 @@
+          - state: equipped-FEET-sealed
+
+- type: entity
+  id: SuitStorageResponsorySecurity
+  parent: SuitStorageBase
+  suffix: Security Responsory Modsuit
+  components:
+    - type: StorageFill
+      contents:
+        - id: OxygenTankFilled
+        - id: ClothingModsuitResponsorySecurity
+        - id: ClothingMaskBreath
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)